src/packages/end-to-end/src/__tests__/ui/selectbox.test.ts

```diff
@@ -0,0 +1,11 @@
+import { test, expect } from '@playwright/test';
+
```

Comment:
So is this and the other test changes.
I needed this fix to see all the options